cmd/calyptia/create.go

```diff
@@ -9,6 +9,7 @@ func newCmdCreate(config *config) *cobra.Command {
 	}
 
 	cmd.AddCommand(
+		newCmdCreateAggregator(config),
```

Comment:
Just a question about the naming the we will adopt, who is involved with the project knows that core is an aggregator, but idk, on my opinion makes sense we treat it as "core" on codebase. Its just an opinion :)

Reply:
Agree, but we still call it "aggregator" on many places of the code. The database table name, the methods on the cloud server, the endpoint, the client, etc.
Let's rename them in another PR.